Input/InputField.module.less

```diff
@@ -101,6 +101,7 @@
 		.input {
 			font-size: @sand-inputfield-small-font-size;
 			height: @sand-inputfield-small-height;
+			line-height: @sand-inputfield-small-height;
```

Comment:
Well... this change could affect tall glyph languages. 3.5 rem -> 2.75 rem is huge.
@juwonjeong Would you please test with our tall glyph? 
@stanca-pop-lgp Rather than changing size, would you please search more why vertical-align: middle doesn't work on iOS and any other workaround?

Reply:
I tested with tall characters and it doesn't seem to be a problem, especially with the merge of #1138.
before: 
![image](https://user-images.githubusercontent.com/63341832/148769296-18856d3f-8058-4622-8d8f-18dc8398a9f3.png)
![image](https://user-images.githubusercontent.com/63341832/148769347-53fffab0-ba7a-4409-9202-4f0089ce2d3f.png)
after:
![image](https://user-images.githubusercontent.com/63341832/148769390-e77b07b3-3575-4cca-8397-2336d91f07e6.png)
![image](https://user-images.githubusercontent.com/63341832/148769423-0de14612-4cf7-45fb-9366-c1439c157449.png)

Reply:
@stanca-pop-lgp Can you answer my question above? 
>@stanca-pop-lgp Rather than changing size, would you please search more about why vertical-align: middle doesn't work on iOS and any other workaround?

Reply:
@seunghoh Yes, sorry, I will continue investigating. I did not find yet "vertical-align: middle" to be a problem on iOS. But will carry on with the research.